backend/api/activity/index.js

```diff
@@ -3,14 +3,43 @@ const express = require('express');
 const activity = express.Router();
 
 /**
- * Finds posts by id
- * @param {Number} id - The id of the post, must be only digits
+ * Returns activity data by id
+ * @param {Number} id - The id of the activity, must be only digits
  */
 activity.route('/:id([0-9]+)')
   .get((req, res) => {
     const statusCode = 200;
+	const shortened = Boolean(req.query.shortened);
```

Comment:
Try to make sure that all code is indented correctly by the linting rules before pushing